src/core/ext/filters/client_channel/lb_policy/xds/xds_override_host.cc

```diff
@@ -355,14 +345,17 @@ void XdsOverrideHostLb::Helper::AddTraceEvent(TraceSeverity severity,
                                                                      message);
 }
 
+static const auto kJsonLoader =
```

Comment:
Please put this inside of the `XdsOverrideHostLbConfig::JsonLoader()` method.  It does not need to be accessed outside of that method, so there's no reason to put it outside.

Let's stick with the same approach that we use everywhere else.  For example:

https://github.com/grpc/grpc/blob/5933b52e1117e0aca1cd3635ef0370577bbff865/src/core/ext/filters/client_channel/lb_policy/outlier_detection/outlier_detection.cc#L1055

Reply:
I don't know of any reason why local static variables would be frowned upon.